src/DotNet.Status.Web.Tests/EmptyTestStartup.cs

```diff
@@ -0,0 +1,5 @@
+namespace DotNet.Status.Web.Tests;
+
+public class EmptyTestStartup
```

Comment:
I see the usage of the class with the same name `public class TestAppFactory : WebApplicationFactory<EmptyTestStartup>` and I assume there is another class with the same name under different namespace. Why do we need to introduce new one? Why not to move existed one into new location..

Reply:
When I did that it wouldn't work. It was complaining that that the class that is used for WebApplicationFactory had to be in the same assembly as the caller

Reply:
have you tried to include that file in the csproj file?

```
  <ItemGroup>
    <Compile Include="path\EmptyTestStartup.cs" />
  </ItemGroup>

```

Reply:
I have not, but this will be consumed as a PackageReference in the future, not as a project referece, so I'm not sure that will work

Reply:
I was wrong, the reason `EmptyTestStartup` can't be in the Testing Utilities project is because it's a library and doesn't have an entry point. `WebApplicationFactory` requires that the class it's using belongs to a project with an entry point

Reply:
After your PR, there will be 2 files with name: `EmptyTestStartup` in the arcade-services repo:
* arcade-services\src\Shared\Microsoft.DotNet.Web.Authentication.Tests\EmptyTestStartup.cs
* arcade-services\src\DotNet.Status.Web.Tests\EmptyTestStartup.cs

What if we just have 1 and add reference to it in both projects:
```
<ItemGroup>
    <Compile Include="..\Shared\Microsoft.DotNet.Web.Authentication.Tests\EmptyTestStartup.cs" />
</ItemGroup>
```